crates/hyperswitch_domain_models/src/payment_methods.rs

```diff
@@ -84,7 +84,7 @@ pub struct PaymentMethod {
         OptionalEncryptableJsonType<api_models::payment_methods::PaymentMethodsData>,
     pub locker_id: Option<VaultId>,
     pub last_used_at: PrimitiveDateTime,
-    pub connector_mandate_details: Option<diesel_models::PaymentsMandateReference>,
+    pub connector_mandate_details: Option<diesel_models::CommonMandateReference>,
```

Comment:
Don't use diesel models in domain model, create a respective domain and convert diesel model into domain model